Lib/asyncore.py

```diff
@@ -34,7 +34,7 @@
 actually using multiple threads. it's really only practical if your program
 is largely I/O bound. If your program is CPU bound, then pre-emptive
 scheduled threads are probably what you really need. Network servers are
-rarely CPU-bound, however.
+rarely CPU bound, however.
```

Comment:
In this case, `CPU-bound` comes after `servers`, so it should not be hyphenated.

["Generally, you need the hyphen only if the two words are functioning together as an adjective before the noun they’re describing. If the noun comes first, leave the hyphen out.."](https://www.grammarly.com/blog/hyphen/)